components/sensor/index.rst

```diff
@@ -170,6 +172,28 @@ Adds a constant value to each sensor value.
 
 Multiplies each value by a constant value.
 
+``min_value``
+*************
+
+Minimum value limit value for the ``min`` value. 
```

Comment:
This will need more text to explain what it does exactly (something like if sensor value is less than X, output X, otherwise sensor value). Important because it's hard to distinguish from the `min` filter otherwise.